bibliopixel/drivers/visualizer.py

```diff
@@ -38,14 +33,15 @@ def __init__(self, num=0, width=0, height=0, pixelSize=15, port=1618, stayTop=Fa
             if "windows" in operating_system:
                 exe_string = "start python"
             elif "darwin" in operating_system:
-                exe_string = "python"
+                exe_string = "python2.7"
                 suffix = "&"
             else:
-                exe_string = "python"
+                exe_string = "python2.7"
```

Comment:
OK - I have also realized that might be an easily-fixable issue with Python
2.x - the relative/absolute imports.

I need to read this: https://www.python.org/dev/peps/pep-0328/



On Mon, Jun 20, 2016 at 6:05 PM, adammhaile <notifications@github.com>
wrote:

> In bibliopixel/drivers/visualizer.py
> <https://github.com/ManiacalLabs/BiblioPixel/pull/53#discussion_r67775706>
> :
>
> >                  suffix = "&"
> >              else:
> > -                exe_string = "python"
> > +                exe_string = "python2.7"
>
> Ewww.... yeah. My main concern was easy cross-platform which is why I used
> tkinter. Sadly no good Python UI's have cropped up in the last 2 years.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/BiblioPixel/pull/53/files/f1413a5dbdcca6ea516504330b5eeec68eb7d502#r67775706>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAPdsr74hGqREeDx7SmIkBH-NAq025KPks5qNw6tgaJpZM4I6Iga>
> .
>



-- 
     /t

http://radio.swirly.com - art music radio 24/7 366/1000